### PR TITLE
Add helpful link

### PR DESCRIPTION
**Pull request by archy-coder**
Added a link to ##Using Verison Control

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [Git Tips](https://github.com/git-tips/tips) - Collection of most commonly used git tips and tricks.
 - [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - Commit Often, Perfect Later, Publish Once: Git Best Practices.
 - [Git Interactive Tutorial](https://learngitbranching.js.org/) - Learn Git in the most visual and interactive way.
+- [Git Cheat Cheets](https://devhints.io/?q=git) - A set of grahical cheat sheets on git
```
Comment (erictleung) [inline README.md:125]: For consistency.

```suggestion
- [Git Cheat Cheets](https://devhints.io/?q=git) - A set of grahical cheat sheets on git.
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for the link. It looks good, just want to keep it consistent with the rest. See comment.

File: README.md (hunk 1 above)
Comment (nhcarrigan) [reply README.md:125]:
```suggestion
- [Git Cheat Cheets](https://devhints.io/?q=git) - A set of graphical cheat sheets on git.
```

Can we correct the typo as well? 🙂

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking the time to find this link and adding it! It'll be useful for others to use when learning git ✨

**Pull request merged**